Bridges/BridgeInterface.php

```diff
@@ -29,8 +29,8 @@ public function getStaticDirectory();
     /**
      * Handle a request using a HttpKernelInterface implementing application.
      *
-     * @param \React\Http\Request $request
-     * @param \PHPPM\React\HttpResponse $response
+     * @param RequestInterface $request
+     * @return ResponseInterface
      */
-    public function onRequest(\React\Http\Request $request, HttpResponse $response);
+    public function onRequest(RequestInterface $request);
```

Comment:
It's possible to change `onRequest` method in `BridgeInterface` to `process` method from `Interop\Http\ServerMiddleware\DelegateInterface`. Yet, I need to wait with this changse since right now `react/http` is using `RequestInterface` and not `ServerRequestInterface` which is needed in `Interop\Http\ServerMiddleware\DelegateInterface`. As soon an update is available, I'll merge it here and change our implementation. It should ship with `react/http: 0.7.0`.